code/game/objects/items/AI_modules/hacked.dm

```diff
@@ -34,3 +34,49 @@
 			law_datum.replace_random_law(laws[1], list(LAW_ION, LAW_HACKED, LAW_INHERENT, LAW_SUPPLIED), LAW_HACKED)
 	return laws[1]
 
+/obj/item/ai_module/malf // Gives syndie laws as well as making AI malf
+	name = "Infected AI Module"
+	desc = "An AI Module, infected with a virus."
+	bypass_law_amt_check = TRUE
+	laws = list("")
+	var/functional = TRUE
```

Comment:
DMDoc this https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md

Reply:
Done